data/lib/src/datasource/authentication_oidc_datasource.dart

```diff
@@ -126,4 +126,16 @@ class AuthenticationOIDCDataSource {
       });
     });
   }
+
+  Future<void> logOut() {
+    return Future.sync(() async {
+      await appAuth.endSession(EndSessionRequest(
```

Comment:
I'm not sure this will work by missing parameters. Let's see the lib's document: https://pub.dev/packages/flutter_appauth#end-session

Reply:
me too, but need to wait to test :)